opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -2969,14 +2969,13 @@ void CpGrid::populateLeafGridCells(Dune::cpgrid::EntityVariableBase<cpgrid::Geom
             const auto& preAdaptFace = face.index();
             int adaptedFace = 0; // It will be rewritten.
             // Face is stored in adapted_faces
-            if ( elemLgrAndElemLgrFace_to_adaptedFace.count({elemLgr, preAdaptFace}) == 1) {
+            if ( elemLgrAndElemLgrFace_to_adaptedFace.find({elemLgr, preAdaptFace}) != elemLgrAndElemLgrFace_to_adaptedFace.end()) {
                 adaptedFace = elemLgrAndElemLgrFace_to_adaptedFace[{elemLgr, preAdaptFace}];
```

Comment:
Please check all similar occurrences (previously `...count()==1`) again. I suggested to used find instead of count to get rid off multiple lookups.
But your change here still has two identical lookups/searches (one in line 2972 (`find`) and one in line 2973 (`operator[]`).

My suggestd change here would be:
```
if ( auto candidate = elemLgrAndElemLgrFace_to_adaptedFace.find({elemLgr, preAdaptFace}); candidate != elemLgrAndElemLgrFace_to_adaptedFace.end()) {
  adaptedFace = *candidate;
}
```
Note that there is only one lookup. In the if-clause we simply dereference the iterator, which is fast.

Reply:
Opps! Sorry, it seems I got half of your suggestion back then. I'll fix it (here and similar places) Thanks!